common/scala/src/main/scala/whisk/http/ErrorResponse.scala

```diff
@@ -57,10 +57,10 @@ object Messages {
     val resourceDoesNotExist = "The requested resource does not exist."
 
     /** Standard message for too many activation requests within a rolling time window. */
-    val tooManyRequests = "Too many requests in a given amount of time for namespace."
+    def tooManyRequests(count: Int, allowed: Int) = s"Too many requests in the last minute (count: $count, allowed: $allowed)."
```

Comment:
Is the time period here configurable via the limits in Ansible? If so, it'd be nice to surface that value here instead of say `in the last minute`.